app/templates/components/forms/user-payment-info-form.hbs

```diff
@@ -0,0 +1,49 @@
+<form class="ui form" {{action 'submit' on='submit'}}> 
+  <h3 class="ui header">
+    {{t 'Payment Information'}}
+  </h3>
+  <div class="field">
+    <label class="required" for="contactName">{{t 'Contact Name'}}</label>
+    {{input type='text' id='contactName' value=(mut userBillingInfo.billingContactName)}}
```

Comment:
read docs on what mut helper does. What you have written won't do anything.

Reply:
Yes, you are right, I have removed it now